.github/workflows/ci.yml

```diff
@@ -20,7 +33,12 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          submodules: true
+          repository: Eppo-exp/eppo-multiplatform
```

Comment:
This one is not an SDK - it is a standalone server that uses `eppo-core` so it won't have the same evaluation testing criteria. Ignore it or another idea to handle?